packages/now-build-utils/src/detect-builders.ts

```diff
@@ -496,7 +496,7 @@ function getMissingBuildScriptError() {
     code: 'missing_build_script',
     message:
       'Your `package.json` file is missing a `build` property inside the `scripts` property.' +
-      '\nMore details: https://vercel.com/docs/v2/platform/frequently-asked-questions#missing-build-script',
+      '\nLearn More: https://vercel.com/docs/v2/platform/frequently-asked-questions#missing-build-script',
```

Comment:
Can you use `link` and `action` here?

Reply:
I'm not sure 🤔 I'm trying to improve it a little step at a time, so I haven't looked at whether we properly handle the `error` object in all the cases here.

Reply:
Ok we can revisit in another PR